tests/unit/fem/test_pa_kernels.cpp

```diff
@@ -353,97 +353,113 @@ TEST_CASE("PA Vector Diffusion", "[PartialAssembly], [VectorPA]")
    }
 }
 
-//test convection
-int dimension;
-
-// Velocity coefficient
 void velocity_function(const Vector &x, Vector &v)
 {
-
-   if (dimension == 2)
+   int dim = x.Size();
+   switch (dim)
    {
-      v(0) = sqrt(2./3.); v(1) = sqrt(1./3.);
+      case 1: v(0) = 1.0; break;
+      case 2: v(0) = x(1); v(1) = -x(0); break;
+      case 3: v(0) = x(1); v(1) = -x(0); v(2) = x(0); break;
    }
+}
+
+void AddConvectionIntegrators(BilinearForm &k, VectorCoefficient &velocity,
+                              bool dg)
+{
+   k.AddDomainIntegrator(new ConvectionIntegrator(velocity, -1.0));
 
-   if (dimension == 3)
+   if (dg)
    {
-      v(0) = sqrt(3./6.); v(1) = sqrt(2./6.); v(2) = sqrt(1./6.);
+      k.AddInteriorFaceIntegrator(
+         new TransposeIntegrator(new DGTraceIntegrator(velocity, 1.0, -0.5)));
+      k.AddBdrFaceIntegrator(
+         new TransposeIntegrator(new DGTraceIntegrator(velocity, 1.0, -0.5)));
    }
-
 }
 
-//Basic unit test for convection
-TEST_CASE("PA Convection")
```

Comment:
There is a lot of changes again in this file initiated by @artv3 - we want to make sure both sides are good with these.

Reply:
The idea is just to extend the "PA Convection" tests to DG, what was done by @artv3 is unchanged.

Reply:
Looks good to me 👍